src/header-navigation/header-navigation.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {getOverrides} from '../helpers/overrides';
+import type {PropsT} from './types';
+import {Root as StyledRoot} from './styled-components';
+
+class HeaderNavigation extends React.Component<PropsT, {}> {
+  static defaultProps = {
+    overrides: {},
+  };
+
+  render() {
+    const {overrides = {}} = this.props;
```

Comment:
Is there a reason to have `{}` as a fallback value if we are defaulting the prop as well?

Reply:
I fixed flow types. No need for that now